flink-runtime-web/src/test/java/org/apache/flink/runtime/webmonitor/handlers/JarHandlerParameterTest.java

```diff
@@ -85,7 +85,7 @@ enum ProgramArgsParType {
     static Time timeout = Time.seconds(10);
     static Map<String, String> responseHeaders = Collections.emptyMap();
 
-    private static Path jarWithManifest;
+    static Path jarWithManifest;
```

Comment:
I don't think this is necessary, we can use `jarWithEagerSink` in `testConfigurationWithEmptySavepointPath` instead of this.

Reply:
Done